cmd/config/init/flags.go

```diff
@@ -19,6 +16,7 @@ func addFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP(FlagNames.HubID, "", StagingHubID, fmt.Sprintf("The ID of the Dymension hub. %s", getAvailableHubsMessage()))
 	cmd.Flags().StringP(FlagNames.RollappBinary, "", "", "The rollapp binary. Should be passed only if you built a custom rollapp")
 	cmd.Flags().StringP(FlagNames.TokenSupply, "", defaultTokenSupply, "The total token supply of the RollApp")
+	cmd.Flags().BoolP(FlagNames.Interactive, "", false, "Run roller in interactive mode")
```

Comment:
should default to true

Reply:
@yishay-dym @shaolin-flow what the default for the docs?